jax/interpreters/xla.py

```diff
@@ -274,12 +292,16 @@ def write(v, node):
   env = {}
   write(core.unitvar, c.Tuple())
   if const_vals:
+    for val in const_vals:
```

Comment:
Nit: the `if const_vals` and `for val in const_vals` are redundant; could remove the `if`.

Reply:
(Because `const_vals` is always a tuple, but may be empty, IIRC.)

Reply:
But we still need the `if const_vals` for the rest of the block (the stuff that was already here)?